tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+envlist = py34, py35, lint
+
+[testenv]
+setenv =
+; both temper-python and XBee modules have utf8 in their README files
+; which get read in from setup.py. If we don't force our locale to a
+; utf8 one, tox's env is reset. And the install of these 2 packages
+; fail.
+    LANG=en_US.UTF-8
+    PYTHONPATH = {toxinidir}:{toxinidir}/homeassistant
+commands =
+     py.test -v --timeout=30 --cov --cov-report= {posargs}
+deps =
+     -r{toxinidir}/requirements_all.txt
+     -r{toxinidir}/requirements_test.txt
+
+[testenv:lint]
+basepython = python3
+commands =
+     flake8
+     pylint homeassistant
+     python script/gen_requirements_all.py validate
+
+[flake8]
```

Comment:
Wouldn't it be better if we put these in [`setup.cfg`](https://github.com/balloob/home-assistant/blob/dev/setup.cfg) ? That way linters in editors will be able to pick it up too